test/writer.test.js

```diff
@@ -144,6 +144,28 @@ describe('writer', function () {
       });
     });
 
+    it('ommits commit category if there was no category defined', function () {
+      var hash = '1234567';
+      var commits = [
+        { type: 'feat', category: 'testing', subject: 'did some testing', hash: hash + 'a' },
+        { type: 'chore', category: '', subject: 'other changes', hash: hash + 'a' }
```

Comment:
No! I just didn't like the fact that two commits would eventually have the same hash...